tests/core/test_simd_float32x4.c

```diff
@@ -5,9 +5,25 @@
 #include <math.h>
 #include <assert.h>
 
+void printFloat(float n) {
```

Comment:
I'm not totally sure, my rough-cut guess is that it's due to wasm not being strictly IEEE 754 compliant with respect to NaN values (https://github.com/WebAssembly/design/blob/master/Semantics.md#floating-point-operators and https://github.com/WebAssembly/design/blob/master/FutureFeatures.md#full-ieee-754-2008-conformance), in particular with respect to NaN signs.